impl/src/main/java/org/jboss/fuse/wsdl2rest/impl/Options.java

```diff
@@ -35,7 +35,10 @@
     @Option(name = "--out", required = true, usage = "Output path for generated artefacts (required)")
     Path outpath;
 
-    @Option(name = "--camel-context", usage = "Path to the generated camel context file name")
+    @Option(name = "--blueprint-context", usage = "Path to the generated blueprint camel context file")
+    Path blueprintContext;
+
+    @Option(name = "--camel-context", usage = "Path to the generated spring camel context file")
```

Comment:
keeping --camel-context instead of renaming to --spring-context is for backward compatibility?

Why having the possibility to set both "--blueprint-context" and "--camel-context"? it is because both are generated every time?

Reply:
Compatibility is of no concern, just yet. You can choose to generate one or the other or both

Reply:
if no compatibility concern, what do you think to name it --spring-context and --blueprint-context or --spring-camel-context and --blueprint-camel-context?
I think it is clearer than --camel-context and --blueprint-context